tests/phpunit/CRM/Core/BAO/CustomValueTableTest.php

```diff
@@ -77,6 +83,7 @@ public function testStoreFile(): void {
    * Test store function for state province.
    */
   public function testStoreStateProvince(): void {
+    $this->hookClass->setHook('hook_civicrm_custom', [$this, 'hook_custom']);
```

Comment:
Do these per-class ones do anything? It looks like the first param shouldn't include the word "hook"? And in any case setUp is setting it.

Reply:
ok have cleaned that up @demeritcowboy